README.md

```diff
@@ -109,10 +109,9 @@ For a new CO make sure
 * application is allowed to be used by CO.
```

Comment:
Note DOI badge doesn't work

Reply:
Works for me: image is blue and links to Zenodo page with latest version.